docs/core/diagnostics/metrics-collection.md

```diff
@@ -0,0 +1,404 @@
+    Allocation Rate (B / 1 sec)                                8,168
+    CPU Usage (%)                                                  0
+    Exception Count (Count / 1 sec)                                0
+    GC Heap Size (MB)                                              2
+    Gen 0 GC Count (Count / 1 sec)                                 0
+    Gen 0 Size (B)                                         2,216,256
+    Gen 1 GC Count (Count / 1 sec)                                 0
+    Gen 1 Size (B)                                           423,392
+    Gen 2 GC Count (Count / 1 sec)                                 0
+    Gen 2 Size (B)                                           203,248
+    LOH Size (B)                                             933,216
+    Monitor Lock Contention Count (Count / 1 sec)                  0
+    Number of Active Timers                                        1
+    Number of Assemblies Loaded                                   39
+    ThreadPool Completed Work Item Count (Count / 1 sec)           0
+    ThreadPool Queue Length                                        0
+    ThreadPool Thread Count                                        3
+    Working Set (MB)                                              30
+```
+
+See the [dotnet-counters](dotnet-counters.md) for more information about the tool or [builtin metrics](available-counters.md) to learn
+more about metrics that are available out of the box in .NET.
+
+### Viewing metrics in Grafana with OpenTelemetry and Prometheus
+
+#### Prerequisites
+
+- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet) or a later version
+
+#### Overview
+
+[OpenTelemetry](https://opentelemetry.io/) is a vendor-neutral open-source project supported by the
+[Cloud Native Computing Foundation](https://www.cncf.io/) that aims to standardize generating and collecting telemetry for
+cloud-native software. The .NET team has designed our built-in platform metric APIs to be compatible with this
+standard to make integration straightforward for any .NET developers that wish to use it. At the time of writing support for
+OpenTelemetry metrics is relatively new but [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-overview)
+and many major APM vendors have endorsed it and have integration plans underway.
+
+This example shows one of the integrations available now for OpenTelemetry metrics using the very popular OSS 
+[Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) projects. The metrics data will flow like this:
+
+1. The .NET metric APIs collect measurements from our sample application
+2. The OpenTelemetry library running inside the same process aggregates these measurements
+3. The Prometheus exporter library makes the aggregated data available via an HTTP
+metrics endpoint. 'Exporter' is what OpenTelemetry calls the libraries that transmit
+telemetry to vendor-specific services.
```

Comment:
"backends" might be a better term https://github.com/open-telemetry/opentelemetry-specification/blob/16da729707e02e83f3a1b25b4faa3094c3f9e29b/specification/library-guidelines.md#sdk-implementation

```suggestion
telemetry to vendor-specific backends.
```